src/js/angular-tooltips.js

```diff
@@ -26,7 +26,7 @@
           , width
           , offsetTop
           , offsetLeft
-          , title = attr.title || ''
+          , title = attr.tooltipTitle || ''
```

Comment:
Here i don't like, sorry,  the tooltip should inherit from title attribute :) or even better attr.tooltipTitle || attr.title i think